Removed padding

vic778 opened the pull request:
- Created  Home page
- Created  About page
- Created CSS file 
- Created Js file 
- [demostration video] https://www.loom.com/share/e379810cc0f64307b535867bdc456e73

vic778 committed:
Removed padding

VanessaAoki commented on MIT.md:
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [Edward Yara]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```
- [ ] Please considering editing this file so it fits your own needs.

VanessaAoki commented on about.html:
```diff
@@ -0,0 +1,266 @@
+          </div>
+        </div>
+        <div class="welcome_to_michigan">
+          <h1 class="Hello">"Hello! Welcome To Michigan"</h1>
+        </div>
+        <div class="about_title">
+          <h1 class="about_title_content">
+            Web Development Course Summit 2021
+          </h1>
```
- [ ] Having more than one `<h1>` is allowed by the HTML specification, but is not considered a best practice, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). For instance, in this case you could consider making the `<h1 class="Hello">` a `<span>` tag. 
- [ ] Keep in mind that there should be only one `<h1>` tag per page, and you have several throughout this page (lines: 190, 219, 234, 242).

VanessaAoki commented on index.js:
```diff
@@ -0,0 +1,79 @@
+  containerSpeaker.className = 'speakers_container';
+  const speakerSection = document.querySelector('.guest_speaker');
+  speakerSection.appendChild(containerSpeaker);
+  for (let i = 0; i < speakerObj.length; i += 1) {
```
- [ ] [OPTIONAL]: Great job implementing the speakers section dynamically with JS. You should consider, for the future, using the `.map` function, you can read more about when and why use map over for loops in this [thread](https://stackoverflow.com/questions/29640254/when-why-to-use-map-reduce-over-for-loops/43883957).

VanessaAoki commented on style.css:
```diff
@@ -0,0 +1,1078 @@
+@import url("https://fonts.googleapis.com/css2?family=DM+Sans:wght@700&family=Heebo:wght@500&family=Karla:wght@800&family=Lato:wght@900&family=Mukta:wght@800&family=Open+Sans:ital,wght@1,300&family=Source+Sans+Pro:wght@200;900&display=swap");
+
+:root {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
```
- [ ] The original design provides you all `font-family` used in this project, so, to match the design better, you should consider using them. 

![image](https://user-images.githubusercontent.com/66967274/142414700-9e4f3372-6420-44b3-96cf-06423818a978.png)

VanessaAoki commented on about.html:
```diff
@@ -0,0 +1,266 @@
+      </div>
+    </section>
+
+    <section class="footerII">
+      <div class="footer_img">
+        <img src="./images/university-of-michigan-seeklogo.com.jpg" width="150" height="30" alt="(U.O.M)_logo">
+      </div>
+      <div class="right_reserved">
+        <h5 class="footer_right_reserved">
+          2021 Web Development Course University Of Michigan, Some Rights Reserved.
+        </h5>
+        <p class="footer_right_reserved_content">
+          In addition, the University Of Michigan reserve the right to terminate the agreement on a long time frame, for years' notice -- half the ordinary notification period.
+        </p>
+      </div>
+    </section>
```
- [ ] To keep HTML semantic, you should consider making this a `<footer>` instead of a `<section>`, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer).
- [ ] In the original design, there are different footers for mobile and desktop versions, in the desktop version the footer has a dark background (just like you did), but in the mobile, it has a white background and the partner section is visible as well, you should consider adding the mobile version, as you can see here:

Behance design (mobile version):
![image](https://user-images.githubusercontent.com/66967274/142415924-81161b58-5390-4498-add5-d0261b067ecc.png)

VanessaAoki commented on index.html:
```diff
@@ -0,0 +1,314 @@
+
+          <ul class="header_txt_date">
+            <li>
+          <h1 class="header_txt_date_item">
+            2021.10.15(THU) ~ 16(FRI)
+          </h1>
```
- [ ] Just like mentioned in the about.html page, you should consider revising all your headings in this page as well.

VanessaAoki commented on index.html:
```diff
@@ -0,0 +1,314 @@
+      <h2 class="partner_title">
+        Partner
+      </h2>
+      <div class="seperatorIII"></div>
```
- [ ] In the desktop version, this separator seems to not be working properly. In the original design it is center, right below the text, so you should consider, instead of having fixed values for margin left and right using something like: `margin: 10px auto` (this means it will have a 10px margin for both top and bottom and auto margin for left and right, keeping it centered. You can read more about margin syntax in [here](https://developer.mozilla.org/en-US/docs/Web/CSS/margin).

VanessaAoki commented on style.css:
```diff
@@ -0,0 +1,1078 @@
+  background-image: url(./images/second_texture.jpg);
+  background-size: cover;
+  height: 500px;
+  margin: 10px;
```
- [ ] In the original design, this section has no margins, it covers the full page width, so, in order to match the original design better, you should consider removing it.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142417342-e9e51811-8109-47c8-99bd-d3d8cfe4cc90.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142417397-b955ccf4-fdf8-467c-a675-47e2112c5cfb.png)

VanessaAoki commented on style.css:
```diff
@@ -0,0 +1,1078 @@
+  margin-top: 50px;
+}
+
+.card_items {
+  width: 200px;
+  text-align: center;
+  background-color: rgba(255, 255, 255, 0.103);
+  margin: 2px;
+  padding: 10px;
+  height: 190px;
+}
```
- [ ] To match the original design better, you should consider centering horizontally this cards elements.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142417956-a3fd1267-d8b4-40dc-9680-202012583457.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142418007-d020dd47-96b2-4a21-8b2e-8b143351ec82.png)


Your design with horizontally centered elements:
![image](https://user-images.githubusercontent.com/66967274/142417919-2b874756-8039-45d1-8ed7-f9d3f9c2221e.png)

VanessaAoki commented on about.html:
```diff
@@ -0,0 +1,266 @@
+        <ul class="authors_profile_container">
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link"
+              ><img
+                src="./images/facebook-square-brands.jpg"
+                class="social_mid"
+                alt="facebook_profile"
+            /></a>
+          </li>
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link"
+              ><img
+                src="./images/twitter-square-brands.jpg"
+                class="social_mid"
+                alt="twitter_profile"
+            /></a>
```
- [ ] Having icons as images can make them look a bit off from your design and the personalization is also very limited. You should consider using something like [fontawesome](https://fontawesome.com/) to add your icons.

How your icons look right now:
![image](https://user-images.githubusercontent.com/66967274/142418481-e676ee9f-4c56-4a9f-a081-96cc27af384b.png)

VanessaAoki commented on style.css:
```diff
@@ -0,0 +1,1078 @@
+
+.second_header_container {
+  padding: 150px;
+  background-image: url(./images/first_bg.jpg);
```
- [ ] [OPTIONAL]: Great job adding this image as a background. You should consider adding a `linear-gradient`, a white one maybe, to help improve readability on mobile version (the bright colors of the image sometimes makes it hard to read the text). You can read more on how apply a filter to a background image [here](https://css-tricks.com/apply-a-filter-to-a-background-image/).

VanessaAoki commented on about.html:
```diff
@@ -0,0 +1,266 @@
+              <a href="#" class="destop_content_list_links">News</a>
+            </li>
+            <li class="destop_content_list campaign_action">
+              <a href="#" class="destop_content_list_links">CC Campaign</a>
```
- [ ] In the original design, this button has a centered orange text, you should consider changing it to match the design better.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142420287-852e9d0d-416e-4c7b-949e-127299cd33e8.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142420246-7afc7713-275b-449a-befa-c913214ad6f2.png)

VanessaAoki reviewed (changes requested):
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @vic778, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Nice presentation video 🎥  

## Required Changes ♻️
_Check the comments under the review._
- [ ] You should make sure that all your README file links works (some such as the presentation and live demo link are currently broken).
- [ ] There is an empty `md` file in your root, you should consider removing it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

![so close](https://c.tenor.com/nMbdSNsg_jsAAAAC/the-simpsons-homer-simpson.gif)

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

vic778 committed:
Adde my Name in the bracket

vic778 committed:
Renamed all h1 to span

vic778 committed:
Added the resentation link

vic778 committed:
Added while background to partner section

vic778 committed:
Added while background to partner section

vic778 committed:
Added the resentation link

vic778 committed:
Renamed all h1 to span

bimbolabuari commented on index.html:
```diff
@@ -0,0 +1,314 @@
+        <header class="header">
+          <nav class="authors_social_link">
+            <ul class="authors_profile_container">
+              <li class="authors_profile">
+                <a href="#" class="authors_profile_link"><img src="./images/facebook-square-brands.jpg" class="social_mid" alt="facebook_profile"></a>
+              </li>
+              <li class="authors_profile">
+                <a href="#" class="authors_profile_link"><img src="./images/twitter-square-brands.jpg" class="social_mid" alt="twitter_profile"></a>
+              </li>
```
Let's fix the social media logo as requested by the previous code reviewer ✊
![Screenshot (312)](https://user-images.githubusercontent.com/66112760/142586421-a8b4ae1c-68f9-492f-aab6-7786f8e3e27f.png)

bimbolabuari reviewed (changes requested):
Hi @vic778 , 

NIce job implementing changes required by the previous code reviewer 🚀  

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

vic778 committed:
Added padding in the compaign class

AshakaE reviewed (approved):
### Hi @vic778  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 46,865.21kb -> 24,515.01kb (47.69%)

/images/mainbg.png -- 39,362.40kb -> 17,906.15kb (54.51%)
/images/airbnb-24.jpg -- 0.81kb -> 0.41kb (48.73%)
/images/-airbnb-50.png -- 2.12kb -> 1.28kb (39.84%)
/images/icons8-airbnb-50.png -- 2.05kb -> 1.28kb (37.35%)
/images/Mitigan4.png -- 72.57kb -> 56.55kb (22.07%)
/images/Workshop.png -- 29.53kb -> 23.26kb (21.22%)
/images/Exhibition.png -- 30.96kb -> 24.40kb (21.19%)
/images/Pattern.png -- 4,041.52kb -> 3,261.50kb (19.3%)
/images/Forum.png -- 21.95kb -> 18.08kb (17.64%)
/images/Lecture.png -- 18.77kb -> 16.32kb (13.07%)
/images/Ignite.png -- 8.12kb -> 7.43kb (8.47%)
/images/airbnb-50.png -- 1.20kb -> 1.14kb (5.29%)
/images/Star_bg.jpg -- 127.61kb -> 120.89kb (5.27%)
/images/Historical_Devs.jpg -- 18.29kb -> 17.57kb (3.94%)
/images/second_Dev_history.jpg -- 15.16kb -> 14.74kb (2.78%)
/images/university-of-michigan-seeklogo.com.jpg -- 36.98kb -> 35.95kb (2.77%)
/images/sep09.jpg -- 2,887.49kb -> 2,822.20kb (2.26%)
/images/hoppwallace.jpg -- 36.51kb -> 36.14kb (1.02%)
/images/kopelmanshirli.jpg -- 35.59kb -> 35.23kb (1.01%)
/images/Bob.jpg -- 39.57kb -> 39.17kb (1.01%)
/images/branchjohn.jpg -- 37.32kb -> 36.95kb (0.99%)
/images/nagarvenky.jpg -- 38.71kb -> 38.39kb (0.84%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

vic778 committed:
Merge pull request #2 from vic778/imgbot

[ImgBot] Optimize images
